packages/host/app/lib/sqlite-adapter.ts

```diff
@@ -215,6 +215,12 @@ export default class SQLiteAdapter implements DBAdapter {
       .replace(/ANY_VALUE\(([^)]*)\)/g, '$1')
       .replace(/CROSS JOIN LATERAL/g, 'CROSS JOIN')
       .replace(/ILIKE/g, 'LIKE') // sqlite LIKE is case insensitive
+      .replace(
+        /jsonb_array_elements_text\(([^)]+)\)\s+AS\s+([^\s]+)/g,
+        (_match, group) => {
+          return `json_each(${group})`;
+        },
+      )
       .replace(/jsonb_array_elements_text\(/g, 'json_each(')
```

Comment:
does your new replacement for `jsonb_array_elements_text` then mean than this original replacement for `jsonb_array_elements_text` is now unnecessary?

Reply:
I think I just made a better solution which I am testing now, so please ignore this for now